src/oneD/Flow1D.cpp

```diff
@@ -32,6 +32,13 @@ const map<string, size_t> componentMap = {
 Flow1D::Flow1D(ThermoPhase* ph, size_t nsp, size_t points) :
     Domain1D(nsp+c_offset_Y, points),
     m_nsp(nsp)
+{
+    warn_deprecated("Flow1D::Flow1D",
+        "To be removed after Cantera 3.2. Use constructor using Solution instead.");
```

Comment:
When deprecating a specific constructor, I think it's helpful to indicate the signature:
```suggestion
    warn_deprecated("Flow1D::Flow1D(ThermoPhase*, size_t, size_t)",
        "To be removed after Cantera 3.2. Use constructor using Solution instead.");
```